app/services/live_config/utils.py

```
@@ -35,6 +36,13 @@ def convert_type(value: Any, target_type: type) -> Any:
             return value
         str_value = str(value) if value else ""
         return [item.strip() for item in str_value.split(",") if item.strip()]
+    elif target_type == dict:
```

Ruff (0.15.14)

[error] Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)